develop/TOOLS/DIS/Developing_DataMiner_protocols_and_Automation_scripts_as_Visual_Studio_solutions.md

```diff
@@ -9,6 +9,14 @@ This section consists of the following topics:
 - [Developing DataMiner protocols as Visual Studio solutions](xref:Developing_DataMiner_protocols_as_Visual_Studio_solutions)
 - [Developing Automation scripts as Visual Studio solutions](xref:Developing_Automation_scripts_as_Visual_Studio_solutions)
 
+## Legacy-style projects vs. SDK-style projects
+
+As from version 2.42, DIS supports SDK-style projects in addition to the legacy-style projects.
+
+At startup, DIS will install new protocol solution and Automation script solution templates that will be used when creating a new protocol or Automation script solution in Visual Studio 2022.
+
+When a legacy-style solution is added, an SDK-style project will also automatically be created (and vice versa).
```

Comment:
I don't really understand this sentence.
When developers work on solutions with the old style, DIS will still generate the legacy-style projects. Only new solutions will be SDK-style.

Reply:
Last sentence has been removed.